CRM/Contribute/Form/ContributionPage/Amount.php

```diff
@@ -752,7 +752,7 @@ public function postProcess() {
             $deletePledgeBlk = FALSE;
             $pledgeBlockParams = [
               'entity_id' => $contributionPageID,
-              'entity_table' => ts('civicrm_contribution_page'),
+              'entity_table' => 'civicrm_contribution_page',
```

Comment:
This will solve the problem but also I just wonder about using `CRM_Contribute_DAO_ContributionPage::getTableName()`

Reply:
I actually find that more confusing when it is only ever going to be the same string. I think it caused quite a few problems in the past & pre-ceded executeQuery handling properly